utils.go

```diff
@@ -27,3 +29,13 @@ func waitForAliveVMM(ctx context.Context, client *Client) error {
 		}
 	}
 }
+
+// envValueOrDefaultInt check if env value exists and returns it or returns default value
+// provided as a second param to this function
+func envValueOrDefaultInt(envName string, def int) int {
+	envVal, err := strconv.Atoi(os.Getenv(envName))
+	if envVal == 0 || err != nil {
+		envVal = def
```

Comment:
I don't know if I like the error being silently switch to the default here. I think we, at minimum, would want to log this back to the user.

Reply:
This function shouldn't produce logs, but if we do want to expose the error it can return `(int, error)`.

Reply:
This is an internal API, so we can change it at will if we need to. We don't have a logger object in most of the contexts in which we call this function, so I'm not sure there's much value in returning the error.

Since this functionality is intended as a convenience for helping developers debug the SDK, it won't be used often and will almost certainly be used in conjunction with inspection of the code, consultation of the docs, etc. So I think we can simply document that the values provided by the environment will be parsed by Atoi() and that errors will be silently swallowed. No need to overthink this.

Reply:
I don't think it is a matter of overthinking but a concern of someone fat fingering a value and it being ignored. I can see some frustration here if a user intended for a value to be used and it turns out we were ignoring it.

I would rather have the error be apparent than the user need to go through code or docs.